kubernetes/README.md

```diff
@@ -4,70 +4,128 @@ This example explains how to deploy the 1Passwrd SCIM bridge on Kubernetes runni
 
 If deploying to the Azure Kubernetes Service, you can refer to our [detailed deployment guide instead](https://support.1password.com/cs/scim-deploy-azure/).
 
-## Create your DNS record
+## Before beginning
+
+There are a few pieces of information you'll want to decide on before beginning the setup process:
+
+* Your SCIM bridge domain name. (example: `op-scim-bridge.example.com`)
+* An email to use for the automatically-created Provision Manager user. (example: `op-scim@example.com`)
```

Comment:
To be crystal clear, I would add an adjective like: `accessible email` as that is a common error I've seen.

Reply:
Added in a better clarification.